libjlc-hls/include/jlc-hls/cmdline.hpp

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2018 Nico Reißmann <nico.reissmann@gmail.com>
```

Comment:
I did not write this file. Could be it that this is just a blunt copy of jlc's cmdline.hpp?

Reply:
Yes, so it is mostly you that has written it